arcgis-ios-sdk-samples/Features/Add delete related features/RelatedFeaturesViewController.swift

```diff
@@ -130,15 +130,18 @@ class RelatedFeaturesViewController: UITableViewController {
         // show progress hud
         SVProgressHUD.show(withStatus: "Applying edits")
         
-        relatedTable.applyEdits { [weak self] (_, error) in
+        relatedTable.applyEdits { [weak self] (results, error) in
             SVProgressHUD.dismiss()
-            
-            if let error = error {
-                // show error
-                self?.presentAlert(error: error)
-            } else {
-                // query to update features
+            if let results = results {
+                if let firstResult = results.first,
+                   firstResult.completedWithErrors {
+                    // The edit fails with error.
+                    self?.presentAlert(error: firstResult.error!)
+                }
+                // Query to update features in the table.
                 self?.queryRelatedFeatures()
```

Comment:
`self` will now be potentially unwrapped multiple times, so it would be best to unwrap it at the beginning of the closure.

Reply:
🤦 🤦  Lemme fix that...